vrs/marten-construction-script.py

```diff
@@ -42,10 +33,8 @@
 logger = logging.getLogger("create_vrs_test_dataset")
 logger.setLevel(logging.INFO)
 
+# Bucket to store temporary data
 TMP_DIR = "gs://gnomad-tmp-4day/vrs"
```

Comment:
since TMP_DIR is only used in the hail init, could delete it here and just use args.tmp_dir within the hail init